mne/channels/tests/test_montage.py

```diff
@@ -27,7 +27,7 @@
 
 from mne.io.kit import read_mrk
 from mne.io import (read_raw_brainvision, read_raw_egi, read_raw_fif,
-                    read_fiducials)
+					read_fiducials)
```

Comment:
please revert these changes.

Reply:
@alexrockhill I don't know what you did to this file but you should reapply a clean patch as you messed up the indentation. It makes the diff not readable.